crates/core/tedge_mapper/src/main.rs

```diff
@@ -115,7 +128,97 @@ async fn main() -> anyhow::Result<()> {
         component.init(&mapper_opt.config_dir).await
     } else if mapper_opt.clear {
         component.clear_session().await
+    } else if component.session_name().eq("tedge-mapper-az") {
+        run_mapper(component.session_name(), config).await
+    } else if component.session_name().eq("tedge-mapper-aws") {
+        run_mapper(component.session_name(), config).await
```

Comment:
On can avoid this by calling `run_mapper()` in the `start()` method of the AWS and Azure mappers.

Reply:
refactored